package.json

```diff
@@ -7,11 +7,9 @@
     "example": "examples"
   },
   "scripts": {
-    "build": "npm run build:css && npm run build:js",
-    "build:js": "browserify site/js/index.js -o site/assets/bundle.js -t [ babelify --presets [@babel/preset-env] --plugins [babel-plugin-transform-class-properties @babel/plugin-proposal-object-rest-spread] ]",
-    "build:css": "postcss site/css/styles.css -o site/assets/styles.css",
-    "test": "eslint site/js/*.js",
-    "start": "node server.js --watch"
+    "build": "npx snowpack@2.14.0",
+    "test": "stylelint site/css/*.css && eslint site/js/*.js",
+    "start": "browser-sync --no-notify"
```

Comment:
Love seeing this simplified server set up. Curious, since you're using snowpack already, why not use `snowpack dev` for the built in server?

Reply:
Oh yeah, I forgot about the built-in server! I'm gonna try this.

Reply:
After playing with this a bit I'm deciding not to include `snowpack dev` for now. It seems really focused on providing on-the-fly compiling on a file-by-file basis, and I was struggling to disable all the built in tooling for that (watching, HMR, etc). Maybe one day though.